packages/workshop-app/app/utils/mdx.tsx

```diff
@@ -1,3 +1,4 @@
+import copyToClipboard from 'copy-to-clipboard'
```

Comment:
I wouldn't use a package for this. Browser support is pretty good to just use the standard API: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText

Reply:
Yes
I forgot for a moment that when the user clicks `copy` the DOM already exist